crypto.c

```diff
@@ -27,13 +27,15 @@
 
 #include <assert.h>
 #include <inttypes.h>
+#include <openssl/sha.h>
```

Comment:
It would be nice not to depend on external libraries. What is this used for?

Reply:
For Mina address generation.  I will switch to an in-built one to be more portable.